### DISCUSSION TITLE
Maintenance respiration Q10 for C4 grasses (Ryan 1991)

### DISCUSSION THREAD

**Discussion by mpaiao** [NGEET/fates#1137] [General]
I was just checking the Ryan (1991) leaf maintenance respiration subroutine, and I noticed that the reference value at 25°C is calculated from the original reference at 20°C using a Q<sub>10</sub> = 1.5

https://github.com/NGEET/fates/blob/5551a70019013c2e73f63c09f3afd786d8151908/biogeophys/FatesPlantRespPhotosynthMod.F90#L2105-L2106

However, when the respiration is calculated for C4 grasses, the actual Q<sub>10</sub> used is 2.0:

https://github.com/NGEET/fates/blob/5551a70019013c2e73f63c09f3afd786d8151908/biogeophys/FatesPlantRespPhotosynthMod.F90#L2121-L2123

Perhaps we would want to use the same value in both cases? I have the impression this is making the reference respiration to be less than intended for C4 grasses...

**Comment by glemieux** [comment]
Consensus is to continue to discuss this and come back to it later.
